Pod/Classes/PURLogStore.h

```diff
@@ -24,7 +25,11 @@ typedef void(^PURLogStoreRetrieveCompletionBlock)(NSArray<PURLog *> *logs);
 
 - (BOOL) registerIndexParameters:(NSArray<NSString *> *)parameters version:(NSString *)version name:(NSString *)name handler:(YapDatabaseSecondaryIndexHandler *)handler NS_SWIFT_NAME(registerIndex(parameters:version:name:handler:));
 - (void) retrieveWith:(NSString *) stringParameter forValue:(NSString *)value usingIndex:(NSString *)index completion:(PURLogStoreRetrieveCompletionBlock)completion NS_SWIFT_NAME(retrieve(with:for:using:completion:));
-- (BOOL) hasIndexWith:(NSString *)name;
```

Comment:
Is there any references to the index feature that you are removing in the Logger? Is safe to remove it?

Reply:
`hasIndexWith` was also introduced by us in a previous PR and was being used in OSTRCSSpanStore (for the traces initiative) and I've already changed it to call `hasExtension`
it's completely safe 👍